### DISCUSSION TITLE
insecure download blocked warning in chrome

### DISCUSSION THREAD

**Discussion by nuweq** [seleniumbase/SeleniumBase#2857] [Q&A] [answered]
Hi, I got this warning from my browser when trying to download a file in Chrome. I use sb.click to click and download the file. How do I bypass this? Can please help with this? Thanks.
![image](https://github.com/seleniumbase/SeleniumBase/assets/37559617/09adf1e1-ecfa-43e6-b392-447cbd76f6ad)

**Comment by mdmintz** [comment, marked as answer]
SeleniumBase uses part of https://stackoverflow.com/a/78065424/7058266 for this:
(`--disable-features=InsecureDownloadWarnings` is already included with SeleniumBase)

You could also try adding: `chromium_arg="--unsafely-treat-insecure-origin-as-secure=*"`, where `*` is your domain.

However, there might be a bug in recent Chromium versions: https://bugs.chromium.org/p/chromedriver/issues/detail?id=4730

**Reply by nuweq** [reply]
Thanks.

**Reply by pikut** [reply]
Unfortunately, from last days this settings it stopped working... Any idea how fix it for now?
